packages/clay-card/src/CardWithInfo.tsx

```diff
@@ -36,6 +29,11 @@ interface IProps {
 	 */
 	disabled?: boolean;
 
+	/**
+	 * Props to add to the dropdown trigger element
+	 */
+	dropDownTriggerProps?: React.HTMLAttributes<HTMLButtonElement>;
```

Comment:
Same as above

> Rather than have just aria labels available to the trigger element, I thought it would be better to provide a more robust API to allow more customization on that element. This allows them to add other accessibility attrs as needed.